mozilla/fx-private-relay
# MPP-2829: Update the Email Tracker Removal Page Styling

This PR fixes MPP-2829
 


# New feature description

This PR updates the styling of the email tracker removal page based on the updated Figma. It makes use of the banner component pieces to create this version, updates the accordion to set focus on initial item and styling is updated. 

https://www.figma.com/file/oc3PM39bpYFBvdJWVyovsq/Tracker-Removal-Report-and-Tracked-Links?type=design&node-id=49-1305&t=UDL2LSelrUV6Ctpt-0 
 
![image](https://user-images.githubusercontent.com/3924990/236582378-cd8cbc0b-833f-4437-a15c-de83774bcda9.png)
 
Not applicable.

# How to test

As per @codemist 's original testing instructions: To test this in local, please paste this mock url which contains the hash data containing the relevant data to create this temporary page: 127.0.0.1:8000/contains-tracker-warning/#%7B%22sender%22%3A%22sender%[40gmail.com](http://40gmail.com/)%22%2C%22received_at%22%3A1682618554000%2C%22original_link%22%3A%22http%3A%2F%[2Fthis.is.a.bad.link.com](http://2fthis.is.a.bad.link.com/)%2F%22%7D

If you hover over the CTA, it should preview a redirect to "[this.is.a.bad.link.com](http://this.is.a.bad.link.com/)".

# Checklist (Definition of Done)
